src/pages/admin/AdminCustomerInstitutionPage.tsx

```diff
@@ -139,94 +140,131 @@ const AdminCustomerInstitutionPage: FC = () => {
-            inputProps={{ 'data-testid': 'customer-institution-archive-name-input' }}
-          />
-          <Field
-            aria-label={CustomerInstitutionFieldNames.CNAME}
-            name={CustomerInstitutionFieldNames.CNAME}
-            label={t('cname')}
-            component={TextField}
-            fullWidth
-            variant="outlined"
-            inputProps={{ 'data-testid': 'customer-institution-cname-input' }}
-          />
-          <Field
-            aria-label={CustomerInstitutionFieldNames.INSTITUTION_DNS}
-            name={CustomerInstitutionFieldNames.INSTITUTION_DNS}
-            label={t('institution_dns')}
-            component={TextField}
-            fullWidth
-            variant="outlined"
-            inputProps={{ 'data-testid': 'customer-institution-institution-dns-input' }}
-          />
-          <Field
-            aria-label={CustomerInstitutionFieldNames.ADMINISTRATION_ID}
-            name={CustomerInstitutionFieldNames.ADMINISTRATION_ID}
-            label={t('administration_id')}
-            component={TextField}
-            fullWidth
-            variant="outlined"
-            inputProps={{ 'data-testid': 'customer-institution-administrator-id-input' }}
-          />
-          <Field
-            aria-label={CustomerInstitutionFieldNames.FEIDE_ORGANIZATION_ID}
-            name={CustomerInstitutionFieldNames.FEIDE_ORGANIZATION_ID}
-            label={t('feide_organization_id')}
-            component={TextField}
-            fullWidth
-            variant="outlined"
-            inputProps={{ 'data-testid': 'customer-institution-feide-organization-id-input' }}
-          />
-          <StyledButtonContainer>
-            <Button color="primary" data-testid="customer-institution-save-button" variant="contained" type="submit">
-              {editMode ? t('common:save') : t('common:create')}
-              {isSaving && (
-                <StyledProgressContainer>
-                  <Progress size={15} thickness={5} />
-                </StyledProgressContainer>
+
+            <Field name={CustomerInstitutionFieldNames.NAME}>
+              {({ field: { name, value }, form: { setFieldValue } }: FieldProps) => (
+                <Autocomplete
+                  disabled={editMode}
+                  options={institutions}
+                  value={institutions.find((i) => i.name === value) ?? null}
+                  getOptionLabel={(option: RecursiveInstitutionUnit) => option.name}
+                  filterOptions={(options: RecursiveInstitutionUnit[], state: any) => {
+                    const inputValue = state.inputValue.toLowerCase();
+                    return options.filter(
+                      (option) =>
+                        option.name.toLowerCase().includes(inputValue) ||
+                        option.acronym.toLowerCase().includes(inputValue)
+                    );
+                  }}
+                  noOptionsText={t('common:no_hits')}
```

Comment:
jeg synes vi burde lage en wrapperkomponent for denne for props som er felles for institusjonsøk. Det er ganske mange fellesprops: f.eks `noOptionsText`, `variant`, `InputProps`, `placeholder`, `getOptionLabel`

Reply:
ja, var inne på det men droppa det fordi denne ble litt mer ulik de andre enn jeg trodde... Men skal gi det et forsøk!